benchbuild/experiment.py

```diff
@@ -160,7 +152,7 @@ def actions(self):
         actions.append(CleanExtra(self))
         return actions
 
-    def sample(self, prj_cls, versions=None):
+    def sample(self, prj_cls: ProjectClass, versions: List[str] = None) -> Iterable[str]:
```

Comment:
R0201 (no-self-use): Method could be a function  <!-- Automatically posted by Lintly -->